llvm/include/llvm/ProfileData/MemProf.h

```diff
@@ -900,9 +900,17 @@ struct LinearCallStackIdConverter {
     Frames.reserve(NumFrames);
     for (; NumFrames; --NumFrames) {
       LinearFrameId Elem =
-          support::endian::readNext<LinearFrameId, llvm::endianness::little>(
-              Ptr);
+          support::endian::read<LinearFrameId, llvm::endianness::little>(Ptr);
+      // Follow a pointer to the parent, if any.
```

Comment:
Maybe point to the comment block further down that describes the radix tree format.